builds/jazz-build-module/scm-module.groovy

```diff
@@ -338,4 +338,27 @@ def getRepoCommitterInfo(commitHash) {
     }
 }
 
+def updateAclPermission(serviceId, policiesList, auth_token, aclUrl) {
```

Comment:
As discussed in https://github.com/tmobile/jazz/pull/676, this has to move out of SCM module.

Reply:
moved to acl-module